### PR TITLE
add Authentication

### PR DESCRIPTION
**Pull request by FengdiZhang**
add Authentication/JWT

### CODE REVIEW

**Comment by LeonYanc** [comment]
need test evidence

**Review by LeonYanc — DISMISSED** [review]

**Comment by FengdiZhang** [comment]
截屏2024-06-21 15 46 21
test screenshot

File: README.md
Comment (LeonYanc) [inline README.md:1]: The readme is good.
But I need a very detailed step by step **manual** for someone who never know your code, but want to just deploy it locally and able to start it up.

**Comment by FengdiZhang** [comment]
README re-edited

**Comment by FengdiZhang** [comment]
截屏2024-06-25 17 57 59
new intermediate table added:

**Review by LeonYanc — APPROVED** [review]

**Pull request merged**